pages/visitor_reservetable.php

```diff
@@ -0,0 +1,270 @@
+<?php
+setTitle("Reserveren");
+
+// Validates all posted data and returns a list of errors.
+// Returns empty array of no errors occured.
+function validateData()
+{
+    $errors = [];
+    if (empty($_POST['InNameOf'])) {
+        $errors[] = "U heeft niet ingevult op wie zijn naam de reservering moet worden ingevuld.";
+    }
+
+    if (empty($_POST['Email'])) {
+        $errors[] = "U moet een email opgeven.";
+    }
+    elseif (!is_email_valid($_POST['Email'])) {
+        $errors[] = "U heeft geen geldig email opgegeven";
```

Comment:
In deze lijst met alle errors ben je af en toe een "." vergeten achter de errors.